daemon/firewall/firewall_manager.go

```diff
@@ -0,0 +1,621 @@
+package firewall
```

Comment:
Please rename `api` and `Api` to `API` where it does not unnecessarily export code. Also, please update where it is mentioned in comments